Validation/EventGenerator/interface/DQMHelper.h

```diff
@@ -17,6 +17,9 @@ class DQMHelper{
  public: 
   DQMHelper(DQMStore::IBooker *i);
   virtual ~DQMHelper();
+
+  MonitorElement* book1dHisto(TString name,TString title,int n,double xmin,double xmax,TString xaxis, TString yaxis);
+  MonitorElement* book2dHisto(TString name,TString title,int nx,double xmin,double xmax,int ny,double ymin,double ymax,TString xaxis, TString yaxis);
```

Comment:
whats the need for these functions? I see lots of string->TString->string conversions that might be better just avoided.